samples/drivers/led_strip/boards/bbc_microbit.overlay

```diff
@@ -8,6 +8,12 @@
 #include <zephyr/dt-bindings/led/led.h>
 
 / {
+	cpus {
```

Comment:
It is better to specify the CONFIG_DELAY_... value when the dts not define cpu clocks.

Reply:
I probably misunderstood your previous remark about creating a conf file. You prefer to remove the 'cpus' section in the overlay file (because it wasn't there before) and have a bbc_micorbit.conf file indicating the CONFIG_DELAY_... values? Correct?

Reply:
Yes.
The devicetree is a description of the hardware configuration.
In other words, the general rule is that it should be changed when the hardware is changed.
In this case, the CPU hardware will not change, so it would be better to configure WS2812_GPIO rather than changing the devicetree.